plugins/secretstores/docker/sample.conf

```
@@ -0,0 +1,11 @@
+# File based Docker Secrets secret-store
+[[secretstores.docker]]
+  ## Unique identifier for the secretstore.
+  ## This id can later be used in plugins to reference the secrets
+  ## in this secret-store via @{<id>:<secret_key>} (mandatory)
+  id = "docker_secretstore"
+
+  ## Default Path to directory where docker stores the secrets file
+  ## Current implementation in docker compose v2 only allows the following
+  ## value for the path where the secrets are mounted at runtime
+  path = "/run/secrets"
```

I would comment this out as it should be the default value.
```suggestion
  # path = "/run/secrets"
```